addons/website_sale/models/res_config.py

```diff
@@ -83,6 +101,16 @@ def set_sale_pricelist_settings(self):
         return self.env['ir.values'].sudo().set_default(
             'sale.config.settings', 'sale_pricelist_setting', sale_pricelist_setting)
 
+    @api.onchange('module_sale_ebay')
+    def _compute_ebay_installed(self):
+        for wizard in self:
+            wizard.ebay_installed = int(self.env['ir.module.module'].search_count(['&', ('state', '=', 'installed'), ('name', '=', 'sale_ebay')]) > 0)
```

Comment:
Actually here you don't have to put the search_count inside the loop. You could extract it and use the value in the loop.